utils/dbhelper.js

```diff
@@ -14,7 +14,7 @@ var DBHelper = function () {
  * @param error Function to call on error
  */
 DBHelper.query = function (sql, parameters, done, error) {
-    if (process.env.USE_SSL) {
+    if (process.env.USE_SSL && process.env.USE_SSL.toLowerCase() !== 'false') {
```

Comment:
This works fine on my Mac (non-ssl) and heroku (ssl), which platform are you using ?